rollup/conf/config.json

```diff
@@ -93,23 +93,16 @@
     "chunk_proposer_config": {
       "propose_interval_milliseconds": 100,
       "max_block_num_per_chunk": 100,
-      "max_tx_num_per_chunk": 100,
       "max_l2_gas_per_chunk": 20000000,
-      "max_l1_commit_gas_per_chunk": 11234567,
-      "max_l1_commit_calldata_size_per_chunk": 112345,
       "chunk_timeout_sec": 300,
-      "max_row_consumption_per_chunk": 1048319,
-      "gas_cost_increase_multiplier": 1.2,
-      "max_uncompressed_batch_bytes_size": 634880
+      "gas_cost_increase_multiplier": 1.2
```

Comment:
What's this parameter for? It doesn't seem to be used. All of these config values should align with what is given in the corresponding structs no?

https://github.com/scroll-tech/scroll/blob/36ed7b438308e3d58047a2c226fe50aa03e47a96/rollup/internal/config/l2.go#L22-L21